File: Py+Algo_Exos/Démos-Exos/Algorithmique/06 - Les répétitions/exo4 - Table multiplication.js
// Demande le chiffre à l'utilisateur
const chiffreUtilisateur = prompt("Pour quel chiffre souhaitez-vous la table de multiplication ?");

// Calcul de la table
for(let multiplicateur = 1; multiplicateur <= 10; multiplicateur++) {
    const produit = multiplicateur * chiffreUtilisateur;
    console.log(multiplicateur + "x" + chiffreUtilisateur + "=" + produit)
}

File: Py+Algo_Exos/Démos-Exos/Algorithmique/06 - Les répétitions/exo4 - Table multiplication.py
# Demande le chiffre à l'utilisateur
chiffreUtilisateur: int = int(input("Pour quel chiffre souhaitez-vous la table de multiplication ? "))

# Calcul de la table
for multiplicateur in range(1, 11):
    produit = multiplicateur * chiffreUtilisateur
    print(str(multiplicateur) + "x" + str(chiffreUtilisateur) + "=" + str(produit))